src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -104,10 +104,10 @@ def get(self, fs_id):
         return self.fs_status(fs_id)
 
     @RESTController.Resource('GET')
-    def clients(self, fs_id):
+    def clients(self, fs_id, suppress_client_ls_errors=True):
```

Comment:
@dparmar18: I don't if it is good to expose this variable to an API where user interacts with and I don't think the user would want to suppress any errors by himself. He would want to see the errors if there are any. Could you add a bit more context tho these changes and also add a tracker to the commit so we have an idea on the problem? Thanks

Reply:
@nizamial09 https://github.com/ceph/ceph/pull/52076#issuecomment-1895662020

#52076 makes it hard to conduct negative tests but I understand it is annoying to get error response if there is no MDS up to serve. So I thought a middle approach that to control this behaviour with a flag, the default bool val being the one set in #52076